compiler/bindings/python/test/ir/dialects_test.py

```diff
@@ -391,3 +391,70 @@ def compilation_info():
     assert compilation_info is not None
     assert compilation_info.lowering_config == lowering_config
     assert compilation_info.translation_info == translation_info
+
+
+@run
+def gpu_target_info_attribute_parsing():
+    mlir_string = """
+    hal.executable private @main_dispatch_0 {
+        hal.executable.variant public @rocm_hsaco_fb
+            target(<"rocm", "rocm-hsaco-fb",
+                {
+                abi = "hip",
+                iree_codegen.target_info = #iree_gpu.target<
+                    arch = "gfx942",
+                    features = "",
+                    wgp = <
+                    compute = fp64|fp32|fp16,
+                    storage = b64|b32|b16|b8,
+                    subgroup = shuffle|arithmetic,
```

Comment:
Could we set the attributes we don't care about to the empty value, e.g., `none`, to keep this minimal?